### DISCUSSION TITLE
Inconsistent terminology - foundations/html_css/flexbox/flexbox-axes.md

### DISCUSSION THREAD

**Discussion by Greydesk** [TheOdinProject/curriculum#25699] [General]
The term used in this section should be AXIS (and it is AXIS in some of the content) instead of AXES. AXES means multiple AX or AXE which are wood cutting tools. AXIS is a single term and it the correct term for this section.

Loving this course so far. I used to do web programming 20 years ago and it is interesting to see what is still the same and what has changed.

**Comment by CouchofTomato** [comment]
Hey @Greydesk 

Do you have a source for this because all resources I have seen state the plural of axis is axes, and sometimes rarely axiis.

**Reply by Greydesk** [reply]
Well, that is interesting. If you look up axis, in the dictionaries, the plural is, indeed, listed as axes. However, if you search up axes, it lists the primary definition as the plural of ax (axe) and the secondary definition as the plural of axis. I stand corrected. I had always assumed that the plural of axis was axis, much like the plural of deer is deer.

**Reply by CouchofTomato** [reply]
Hey @Greydesk 

Someone else made an issue about this a couple of days ago which is why it's fresh in my mind. Prior to that I'd have always pronounced is as axiis myself 🤷

**Comment by CouchofTomato** [comment]
Closing this for now.
